xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/PanelSheet.xml

```diff
@@ -38,11 +38,7 @@
   <hidden>true</hidden>
   <content>{{velocity output="false"}}
 #macro(displayPanelProperty $obj $propName)
-  ; #if ($xcontext.action == 'inline')
-    {{html}}&lt;label for="${class.getName()}_${obj.number}_${propName}"&gt;$services.localization.render("${class.getName()}_${propName}")&lt;/label&gt;{{/html}}
-  #else
-    $services.localization.render("${class.getName()}_${propName}")
-  #end
+  ; &lt;label#if ($xcontext.action == 'edit') for="${class.getName()}_${obj.number}_${propName}"#end&gt;$services.localization.render("${class.getName()}_${propName}")&lt;/label&gt;
```

Comment:
I think changing the if from inline to edit introduced the following regression: https://ci.xwiki.org/job/XWiki/job/xwiki-platform/job/master/1610/testReport/junit/org.xwiki.test.webstandards/CustomDutchWebGuidelinesValidationTest/Platform_Builds___main___flavor_test_webstandards___Build_for_Flavor_Test___Webstandards___Validating_HTML5_Dutch_Web_Guidelines_validity_for__http___127_0_0_1_8080_xwiki_bin_inline_Panels_QuickLinks__executed_with_credentials_Admin_admin/

I'll fix it by checking also the inline: was there a specific reason why you changed this condition?

Reply:
Oh. 
I primarily followed the Help.Applications.Contributors.Code.ContributorsSheet template which is pretty similar to Panels.